### PR TITLE
Translated rendering element page

### PR DESCRIPTION
**Pull request by AbhishekCode**
Translated "rendering element" page. Please check .

### CODE REVIEW

File: content/docs/rendering-elements.md [hunk 1]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
-
-An element describes what you want to see on the screen:
+एलिमेंट्स एक रिएक्ट ऐप्स के सबसे छोटे बिल्डिंग ब्लॉक हैं।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `एलिमेंट्स एक रिएक्ट` => `एलिमेंट्स रिएक्ट`

File: content/docs/rendering-elements.md [hunk 2]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
+एक एलिमेंट वर्णन करता है कि आप स्क्रीन पर क्या देखना चाहते हैं:      
+
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+ब्राउज़र DOM एलेमेंट के विपरीत, React एलेमेंट्स एक सादा ऑब्जेक्ट है और इन्हें बनाना आसान है। React DOM React एलेमेंट्स के अनुसार DOM को अद्यतन बनाए रखता है।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `एक सादा`  sounds a bit odd.
Let's change this to `प्लेन`

File: content/docs/rendering-elements.md [hunk 3]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
->**Note:**
+>**टिप्पणी:**
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `टिप्पणी` => `ध्यान दें`

File: content/docs/rendering-elements.md (hunk 2 above)
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `एलेमेंट` => `एलेमेंट्स`

Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `DOM को अद्यतन बनाए रखता है।` => `DOM को अपडेट रखता है।`

File: content/docs/rendering-elements.md [hunk 4]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+> आप एलेमेंट्स को बहुचर्चित "कौम्पोनॅन्टस" समझ कर भ्रमित हो सकते है, हम "कौम्पोनॅन्टस" के बारे में [अगले भाग](/docs/components-and-props.html) में बतायेंगे। एलेमेंट्स वो है जिनका उपयोग करके एक कौम्पोनॅन्ट को बनाया जाता। आगे बढ़ने से पहले इस भाग को अवश्य पढ़े।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `हम "कौम्पोनॅन्टस" के बारे में` => `हम कौम्पोनॅन्टस के बारे में`

Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `आगे बढ़ने से पहले इस भाग को अवश्य पढ़े।` => `आगे बढ़ने से पहले इस भाग को पड़ने की सलाह देते हैं ।`

File: content/docs/rendering-elements.md [hunk 5]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
+## DOM में एक Element बनाना {#rendering-an-element-into-the-dom}
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `DOM में एक Element बनाना` => `DOM में एक एलिमेंट रेंडर करना`

File: content/docs/rendering-elements.md [hunk 6]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
-Let's say there is a `<div>` somewhere in your HTML file:
+मान लीजिए की आपकी HTML फ़ाइल में कही एक `<div>` है:
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `कही` => `कहीं`

File: content/docs/rendering-elements.md [hunk 7]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
+एक React एलेमेंट को DOM नोड में दर्शाने के लिए, आपको इन दोनो को `ReactDOM.render()` में भेजना पड़ेगा:
 
 `embed:rendering-elements/render-an-element.js`
 
 [](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
-
-## Updating the Rendered Element {#updating-the-rendered-element}
+ये "नमस्ते दुनिया" को पेज पर दर्शाता है
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: I think we should not translate this `"नमस्ते दुनिया"` as in the code same its `Hello world`.

Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: Add a line break after line 44

File: content/docs/rendering-elements.md [hunk 8]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
 `embed:rendering-elements/render-an-element.js`
 
 [](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
-
-## Updating the Rendered Element {#updating-the-rendered-element}
+ये "नमस्ते दुनिया" को पेज पर दर्शाता है 
+## पहले से बने एलेमेंट को अद्यतन करना {#updating-the-rendered-element}
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `अद्यतन` => `अपडेट`

File: content/docs/rendering-elements.md [hunk 9]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
-React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object). Once you create an element, you can't change its children or attributes. An element is like a single frame in a movie: it represents the UI at a certain point in time.
+React एलेमेंट्स  [immutable](https://en.wikipedia.org/wiki/Immutable_object) होते है। एक बार एलेमेंट के बनने के बाद, आप उसके स्वरूप और गुड में परिवर्तन नहीं कर सकते। एक एलेमेंट एक चलचित्र के एक दृश्य के समान है, ये UI को एक निश्चित समय के लिए दर्शाता है।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `immutable` => `अचल` ?

Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `स्वरूप और गुड` => `चिल्ड्रन या ऐट्रिब्यूट्स`

File: content/docs/rendering-elements.md [hunk 10]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
-Consider this ticking clock example:
+इस टिकिंग घड़ी उदाहरण पर विचार करें:
 
 `embed:rendering-elements/update-rendered-element.js`
 
 [](codepen://rendering-elements/update-rendered-element)
 
-It calls `ReactDOM.render()` every second from a [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
+ये `ReactDOM.render()` को प्रत्येक सेकंड [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback से कोल करता है।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `कोल` => `कॉल`

File: content/docs/rendering-elements.md [hunk 11]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
->**Note:**
+>**टिप्पणी:**
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `टिप्पणी` => `ध्यान दें`

File: content/docs/rendering-elements.md [hunk 12]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 >
->In practice, most React apps only call `ReactDOM.render()` once. In the next sections we will learn how such code gets encapsulated into [stateful components](/docs/state-and-lifecycle.html).
+>व्यवहार में, ज़्यादातर React apps `ReactDOM.render()` को एक ही बार कोल करते है. अगले भाग में हम ये सीखेंगे कि कैसे इस तरह के कोड को [stateful components](/docs/state-and-lifecycle.html) में encapsulate करते है।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `कोल` => `कॉल`

Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `encapsulate` => `एन्काप्सुलेट`

File: content/docs/rendering-elements.md [hunk 13]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 >
->We recommend that you don't skip topics because they build on each other.
+>हम अनुशंसा करते हैं कि आप विषयों को छोड़ें नहीं क्योंकि वे एक-दूसरे पर निर्माण करते हैं।
 
-## React Only Updates What's Necessary {#react-only-updates-whats-necessary}
+## React वही अपडेट करता है जो आवश्यक है {#react-only-updates-whats-necessary}
 
-React DOM compares the element and its children to the previous one, and only applies the DOM updates necessary to bring the DOM to the desired state.
+React DOM एलेमेंट्स और उसके चिल्डरेंस की तुलना उसके पहले की अवस्था से करता है और केवल उन्ही चीज़ों में परिवर्तन करता जिससे की DOM वांछित स्थिति में आ जाय।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `वांछित` sounds too complex

File: content/docs/rendering-elements.md [hunk 14]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
-You can verify by inspecting the [last example](codepen://rendering-elements/update-rendered-element) with the browser tools:
+आप ब्राउज़र टूल की सहायता से [अंतिम उदाहरण](codepen://rendering-elements/update-rendered-element) का निरीक्षण करके सत्यापित कर सकते हैं:
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: `टूल` => `टूलस`

File: content/docs/rendering-elements.md [hunk 15]
```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
...
 
 ![DOM inspector showing granular updates](../images/docs/granular-dom-updates.gif)
 
-Even though we create an element describing the whole UI tree on every tick, only the text node whose contents has changed gets updated by React DOM.
+भले ही हम हर टिक पर पूरे UI ट्री का वर्णन करने वाला एक एलेमेंट बनाते हैं, लेकिन केवल Text नोड जिसकी सामग्री बदल गई है वह React DOM द्वारा अपडेट हो जाता है।
 
-In our experience, thinking about how the UI should look at any given moment rather than how to change it over time eliminates a whole class of bugs.
+हमारे अनुभव में, यह सोचना कि UI को किसी भी क्षण को कैसे दिखना चाहिए, न कि इसे समय के साथ कैसे बदलना है, बहुत सारी बग की संभावनाओं को दूर कर देता है।
```
Comment (arshadkazmi42) [inline content/docs/rendering-elements.md]: This does not shows correct meaning, can you rephrase it to show similar meaning as in english

**Comment by AbhishekCode** [comment]
Alright working on it

**Comment by arshadkazmi42** [comment]
@AbhishekCode any updates on this?

**Comment by arshadkazmi42** [comment]
@AbhishekCode Any updates on this? Let us know if you need any help from our end

**Comment by AbhishekCode** [comment]
@arshadkazmi42  Please review, updated

File: content/docs/rendering-elements.md (hunk 2 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: `एलेमेंट्स` => `एलिमेंट्स`

actually this sounds more relevant, can you change this everywhere

File: content/docs/rendering-elements.md (hunk 4 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: You missed this

File: content/docs/rendering-elements.md (hunk 12 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: `कॉल करते है.` => `कॉल करते है।`

File: content/docs/rendering-elements.md (hunk 13 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: `जिससे की DOM अप्डेट हो जाए` => `जिससे की DOM अपने चाहे state में आ जाए`

**Comment by AbhishekCode** [comment]
@arshadkazmi42 Hi, Updated as per comment. Please check and let me know. Thanks.

File: content/docs/rendering-elements.md (hunk 9 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: `एलेमेंट्स  [immutable]` => `एलेमेंट्स [immutable]`

**Comment by arshadkazmi42** [comment]
@AbhishekCode there are still a couple of pending feedbacks, check above this comment, all  the feedbacks which are not resolved.

File: content/docs/rendering-elements.md (hunk 7 above)
Comment (AbhishekCode) [reply content/docs/rendering-elements.md]: Right

**Comment by AbhishekCode** [comment]
> Still some feedbacks are pending

Okay Please check now.

File: content/docs/rendering-elements.md (hunk 4 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: `बनाया जाता। आगे बढ़ने` => `बनाया जाता है। हम आपको आगे बढ़ने`

File: content/docs/rendering-elements.md (hunk 2 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: You missed this one

File: content/docs/rendering-elements.md (hunk 7 above)
Comment (arshadkazmi42) [reply content/docs/rendering-elements.md]: `पर दर्शाता है ` => `पर दर्शाता है।`

**Comment by saranshkataria** [comment]
@AbhishekCode are you still working on this?

**Comment by saranshkataria** [comment]
@arshadkazmi42  should we close this out since there has not been activity for a while on this now?

**Comment by arshadkazmi42** [comment]
@saranshkataria I was planning on working on the inactive PRs myself and you can review it, like this we will be able to merge all the PRs and existing efforts will not be wasted.
Thoughts?

**Comment by saranshkataria** [comment]
That would depend if the original contributor allowed access to change things on this branch, else we will have to copy the work and start working on it as a new PR. I can start doing that too

**Comment by arshadkazmi42** [comment]
I think we would be having access as its foked from this. If we dont have acces, then copies the changed and creating new PR is also a good option

**Comment by saranshkataria** [comment]
Picking this one up, closing this PR and creating a new one since package json has been changed in this one too and that was not needed. @arshadkazmi42 moving this to #41 and fixing the 3 remaining ones as a new commit
